users/details/script.js

```diff
@@ -350,6 +371,28 @@ function generateReadableDateFromSecondsTimeStamp(timeStamp) {
   //created function for readable date format
   return new Date(timeStamp * 1000).toDateString(); // new function because we are getting the value in seconds and not milliseconds
 }
+console.log(generateReadableDateFromSecondsTimeStamp('1688445662'));
+
+function generateDaysToGo(dateStr, status) {
+  const inputDate = new Date(dateStr);
+
+  const now = new Date();
+  const offset = 330 * 60 * 1000;
+  const currentDate = new Date(now.getTime() + offset);
+  const diff = inputDate - currentDate; // Calculates the difference in milliseconds
+  if (diff <= 0 && status == 'COMPLETED') {
+    return 'Task Completed Within Deadline';
+  } else if (diff <= 0) {
+    return 'Deadline Passed'; // Due date is in the past
+  } else if (diff > 0 && diff < 24 * 60 * 60 * 1000) {
+    return 'Less Than a Day Remaining'; // Less than a day remaining
```

Comment:
Less than a Day is remaining.

with Yellow font color

Reply:
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/31709147/7e88de8e-264b-48ca-8cba-026be9ba6b73)

For some reason, I am finding **Yellow** to be a bit difficult to read, I tried **orange** as well, I think it provides better readability, but I know you suggested **yellow** as it is supposed to be a **warning sign**.
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/31709147/86bfadd4-fe7b-4d3d-99b0-1ed77ea850f4)

Please let me know which one should I go ahead with.

Reply:
Currently I have put yellow.